### PR TITLE
Starter notebook with gdrive as temp directory for colab

### PR DESCRIPTION
**Pull request by bamtak**
Starter notebook with GoogleDrive as temp directory for Colab
English to Yoruba improved baseline
Yoruba to English baseline
Notebook with Tensorboard for visualization

### CODE REVIEW

File: starter_notebook_gdrive.ipynb [hunk 1]
```diff
@@ -0,0 +1,3425 @@
+{
```
[the remark this answers is not in the document]
Comment (bamtak) [reply starter_notebook_gdrive.ipynb:1]: starter_notebook_gdrive.ipynb it is a standard notebook like the starter_notebook but using google drive as temporary storage during training, this ensures the ability to continue from the last checkpoint when the connection is lost on colab.

starter_notebook_retrain_from_checkpoint.ipynb is not a complete notebook like starter_notebook, it assumed that processed train and test dataset are already copied to google drive as well as last checkpoint you want to train from.

[the remark this answers is not in the document]
Comment (juliakreutzer) [reply starter_notebook_gdrive.ipynb:1]: Ah, that makes sense. This is a great addition.

@jaderabbit should we have the gdrive checkpoint saving in the starter notebook in general?

**Comment by bamtak** [comment]
@juliakreutzer @jaderabbit Can we finalize this request, I think all the necessary changes have been effected.

File: en-yo/jw300-baseline-improve/README.md [hunk 2]
```diff
@@ -48,3 +48,7 @@ Example 4
 Tokenization | BLEU dev | BLEU test
 --- | --- | ---
 BPE| 31.03 | 38.62
+
+## BPE
+
+- [Link to google drive folder with bpes](https://drive.google.com/open?id=1-63nP5CqUa8JRGLRxRY6vKZd9jgM-Y9)
```
Comment (juliakreutzer) [inline en-yo/jw300-baseline-improve/README.md:54]: The link doesn't work for me, can you please double-check it's correct? Thanks!

Comment (bamtak) [reply en-yo/jw300-baseline-improve/README.md]: Please check now, updated.

**Pull request merged**